### ISSUE TITLE
feat: Analytics api

### ISSUE THREAD

**Issue by NathanWalker** [EddyVerbruggen/nativescript-plugin-firebase#125]
Didn't look like an api was exposed for the Analytics yet although analytics are integrated in.
Just wanted to open this for tracking an contributions here (I would like to at some point on this).

iOS: https://firebase.google.com/docs/analytics/ios/start

Android: https://firebase.google.com/docs/analytics/android/start/

Obviously currently since the native SDK's are loaded, you *could* code against these native apis as it stands right now. It will be nicer to provide consistent api via this plugin for both iOS and Android though.

**Comment by EddyVerbruggen** [comment]
Hey Nathan,

Good idea! It's loaded with Firebase Core, so why not use it :)

Something like this would be nice:

```js
firebase.analytics.logEvent(name:String, parameters:[{key:String, value:String}]);
firebase.analytics.setUserProperty(name:String, value:String);
```

**Comment by EddyVerbruggen** [comment]
Just released 3.6.0 which includes the new Analytics API, see [the doc](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/master/docs/ANALYTICS.md).
